howto_io.md

```diff
@@ -0,0 +1,268 @@
+# How-to: NumPy I/O
+- [Read a .csv or other text file with no missing values](#read-a-csv-or-other-text-file-with-no-missing-values)
+- [Read a .csv or other text file with missing values](#read-a-csv-or-other-text-file-with-missing-values)
+  * [Non-whitespace delimiters](#non-whitespace-delimiters)
+    + [Masked-array output](#masked-array-output)
+    + [Array output](#array-output)
+    + [Array output, specified fill-in value](#array-output-specified-fill-in-value)
+  * [Whitespace-delimited](#whitespace-delimited)
+- [Read a file in .npy or .npz format](#read-a-file-in-npy-or-npz-format)
+- [Write to a file to be read back by NumPy](#write-to-a-file-to-be-read-back-by-numpy)
+  * [Binary](#binary)
+  * [Human-readable](#human-readable)
+  * [Large arrays](#large-arrays)
+- [Read an arbitrarily formatted binary file ("binary blob")](#read-an-arbitrarily-formatted-binary-file-binary-blob)
+- [Write or read very large arrays](#write-or-read-very-large-arrays)
+- [Write files for reading by other (non-NumPy) tools](#write-files-for-reading-by-other-non-numpy-tools)
+- [Write or read a JSON file](#write-or-read-a-json-file)
+- [Save/restore using a pickle file](#saverestore-using-a-pickle-file)
+- [Convert from a pandas DataFrame to a NumPy array](#convert-from-a-pandas-dataframe-to-a-numpy-array)
+- [Save/restore using `numpy.tofile` and `numpy.fromfile`](#saverestore-using-numpytofile-and-numpyfromfile)
+
+## Read a .csv or other text file with no missing values
+
+Use [numpy.loadtxt](https://numpy.org/doc/stable/reference/generated/numpy.loadtxt.html).
+
+## Read a .csv or other text file with missing values
+
+Use
+[numpy.genfromtxt](https://numpy.org/doc/stable/user/basics.io.genfromtxt.html).
+
+``numpy.genfromtxt`` will either return a
+[masked array](https://numpy.org/doc/stable/reference/maskedarray.generic.html)
+masking out missing
+values (if ``usemask=True``) or will fill in the missing value with the value
+specified in ``filling_values`` (default is ``np.nan`` for float, -1 for int).
+
+### Non-whitespace delimiters

```

Comment:
I think this should all be under a top-level "Reading formatted text files", with an introduction narrative including the CSV acronym,  loadtxt and genfromtext, with a sentence about when to use each. Then this section's title should be "Missing Values: CSV files" and the next section "Missing Values: other formats"

Reply:
Done.